### ISSUE TITLE
Allow redis database prefixes

### ISSUE THREAD

**Issue by pavinthan** [tlaverdure/laravel-echo-server#400]
Allow redis database prefixes by default because now Laravel shift with redis prefix by defautlt https://github.com/laravel/laravel/pull/4986

**Comment by shijunti19** [comment]
laravel-echo-server.json
```js
"databaseConfig": {
		"redis": {
			"port": "",
			"host": "",
			"password": "",
			"keyPrefix":"_database_"
		},
```
But it's still useless. The whole system is not well written by the government.

use Illuminate\Broadcasting\Channel;
use Illuminate\Broadcasting\PrivateChannel;
use Illuminate\Broadcasting\PresenceChannel;

All of these files need to be modified and prefixed.

So you have to delete the prefix to fix it.

I was poor and wasted a precious day.

**Comment by kkomelin** [comment]
Spent a few hours debugging this issue after upgrading Dev server to the latest Laravel version.
Found the problem by enabling `devMode` for Laravel Echo Server on two sites (one with old Laravel version and one with the latest one).

Before upgrade:
> Channel: private-conversation.19  
Event: App\Events\MessageSent

After upgrade:
> Channel: laravel_database_private-conversation.19
Event: App\Events\MessageSent

As you may see, the site after upgrade uses strange channel name `laravel_database_private-conversation.19`. It's because of a new Redis prefix Laravel received some time ago.
In fact, Redis prefixes were introduced by this PR https://github.com/laravel/laravel/pull/4982

Temporary workaround is to remove the prefix by adding this setting:
```ini
REDIS_PREFIX=""
```
to the .env config.

Hope it helps someone.

**Comment by tlaverdure** [comment]
Closed by #464. Please also upgrade Laravel Framework to 6.5.2

**Comment by pavinthan** [comment]
thanks @tlaverdure
